tools/build_defs/framework.bzl

```diff
@@ -443,6 +443,7 @@ def _copy_deps_and_tools(files):
     return lines
 
 def _symlink_contents_to_dir(dir_name, files_list):
+    files_list = collections.uniq(files_list)
```

Comment:
Add a comment here?

"It is possible that some duplicate libraries will be passed as inputs to cmake_external or configure_make. Filter duplicates out here."